### PR TITLE
Updated to latest Pega configs and udpated eslint

### PR DESCRIPTION
**Pull request by niallriddell**

### CODE REVIEW

**Review by tumms2021389 — APPROVED** [review]
Looks good to me

just added a small comment

File: sdk-config.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "comment_serverConfig": "serverConfig is the block for SDK Content Server config entries",
     
     "infinityRestServerUrl_comment": "Full path to Infinity REST server",
-    "infinityRestServerUrl": "https://localhost:1080/prweb",
+    "infinityRestServerUrl": "https://lab0427.lab.pega.com/prweb",
```
Comment (tumms2021389) [inline sdk-config.json:24]: change this to localhost url

File: package.json [hunk 2]
```diff
@@ -37,7 +37,7 @@
     "@lion/radio-group": "^0.21.1",
     "@lion/select": "^0.17.1",
     "@lion/textarea": "^0.16.1",
-    "@pega/constellationjs": "SDK-23.1.1",
+    "@pega/constellationjs": "~23.1.1",
```
[the remark this answers is not in the document]
Comment (tumms2021389) [reply package.json:40]: please uninstall constellationjs package from dependencies

**Review by tumms2021389 — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (niallriddell) [reply package.json]: All done.

File: sdk-config.json (hunk 1 above)
Comment (niallriddell) [reply sdk-config.json]: Done

**Review by tumms2021389 — APPROVED** [review]

**Pull request closed without merging**